tests/integration/gcp.py

```diff
@@ -519,22 +515,31 @@ def create_vm(self):
             },
         }
 
-        operation = self._compute.instances().insert(project=self.project, zone=self.zone, body=config).execute()
+        if self.config['secureboot']:
+            config.update(
```

Comment:
```
config['shielded_instance_config'] = {
  "enable_secure_boot": True,
  "enable_integrity_monitoring": True,
  "enable_vtpm": True
}
```

Reply:
What's wrong with the update function, the code is a lot more readable with the update function?

Reply:
It's not the way how it's done in Python. From an programmer perspective this function costs more than simply assigning values. The updates() function should only be used, if you want to merge two dictionaries into a single one. In fact, what you are doing is creating a temporary third dictionary, which contains another dictionary as a value in order to merge the third one with the first dict. 

What you really want to do is a simple value assignment within your first dictionary (technically just linking two memory blocks with each other). Although, the value itself may be an dictionary again, it is recommended to directly assign values, which is a simple task from a CPU perspective compared to the update function which compares dictionaries key one by one with each others and then tries to merge the values accordingly.